client/state/action-types.js

```diff
@@ -585,6 +585,9 @@ export const READER_UPDATE_NEW_POST_EMAIL_SUBSCRIPTION = 'READER_UPDATE_NEW_POST
 export const RECEIPT_FETCH = 'RECEIPT_FETCH';
 export const RECEIPT_FETCH_COMPLETED = 'RECEIPT_FETCH_COMPLETED';
 export const RECEIPT_FETCH_FAILED = 'RECEIPT_FETCH_FAILED';
+export const REWIND_STATUS_ERROR = 'REWIND_STATUS_ERROR';
```

Comment:
...Further to this, we _do_ get useful 4XX responses:
<img width="446" alt="screen shot 2017-06-06 at 13 55 55" src="https://cloud.githubusercontent.com/assets/7767559/26830188/ed9808c4-4abf-11e7-892f-b5e9a29d5301.png">